checkov/secrets/scan_git_history.py

```diff
@@ -26,7 +27,7 @@ def get_commits_diff(root_folder: str) -> Dict[str, Dict[str, str]]:
     except Exception as e:
         logging.error(f"Folder {root_folder} is not a GIT project {e}")
         return commits_diff
-    commits = list(repo.iter_commits(repo.active_branch))
+    commits = list(repo.iter_commits(repo.active_branch, max_count=3))
```

Comment:
accidentally added `max_count` here? 😄

Reply:
yes just for testing - will remove before I merge